store/sqlstore/channel_store.go

```diff
@@ -2545,7 +2545,8 @@ func (s SqlChannelStore) MigrateChannelMembers(fromChannelId string, fromUserId
 		return nil, nil
 	}
 
-	for _, member := range channelMembers {
+	for i := range channelMembers {
+		member := channelMembers[i]
```

Comment:
Nit: I'm not sure if this makes the code more readable. 1/5 that we just stay consistently with the rest of the code and use 
```suggestion
		member := member
```

Reply:
Originally I had kept it `member := member`. But it was @streamer45's suggestion to do it like this. I am 0/5 on either. You guys can decide. :P

Reply:
Yes, this was my suggestion.

I'm failing to see the readability issue of using the index to make a copy of the data we need.

The previous code was creating a `member` variable in the for statement which we would never use. We would then shadow that by making another variable with the same name and then a second copy of the same data.
I would argue that it's better to avoid the bug than to fix it and that is what the suggestion does in my opinion.

As far as the consistency issue I think we are being consistent with the data we are dealing with. These were the only two similar cases where it was straightforward to make a single copy.